pydantic_ai_slim/pydantic_ai/_agent_graph.py

```diff
@@ -206,6 +215,12 @@ async def _sys_parts(self, run_context: RunContext[DepsT]) -> list[_messages.Mod
                 messages.append(_messages.SystemPromptPart(prompt))
         return messages
 
+    async def _instructions(self, run_context: RunContext[DepsT]) -> str:
```

Comment:
Is it really better to always return a str rather than a `str | None`? My gut reaction is that returning `None` if there are no instructions or instruction functions is probably better.